src/util/coordinate_calculation.cpp

```diff
@@ -322,7 +396,8 @@ double degreeToPixel(FloatLatitude lat, unsigned zoom)
 }
 
 // Converts a WMS tile coordinate (z,x,y) into a wsg84 bounding box
-void xyzToWSG84(const int x, const int y, const int z, double &minx, double &miny, double &maxx, double &maxy)
+void xyzToWSG84(
+    const int x, const int y, const int z, double &minx, double &miny, double &maxx, double &maxy)
 {
```

Comment:
I built a local LLVM 3.8, can use that for clang-format'ing.

In case you also want one, here's how to build it: https://gist.github.com/daniel-j-h/8ee8c511f89ebe80821b (needs some adaption on OSX, I think there's no sha256sum and no realpath, otherwise should be fine)